server/routes/bookedtoday/arrivals/courtreturns/confirmCourtReturnAddedToRollController.test.ts

```diff
@@ -36,6 +40,19 @@ describe('confirmCourtReturnAddedToRollController', () => {
         .expect('Location', '/autherror')
     })
 
+    it('should redirect to /duplicate-booking-prevention if arrival already confirmed', () => {
+      lockManager.getLockStatus.mockResolvedValue(true)
+
+      app = appWithAllRoutes({
```

Comment:
shouldn't need to recreate the app here? you have a reference to the lock manager that it's using any way?

Reply:
this applies to the other tests too

Reply:
agreed